opsdroid/connector/rocketchat/__init__.py

```diff
@@ -96,10 +96,10 @@ def build_url(self, method):
         """
         if response["messages"]:
             message = Message(
-                response["messages"][0]["msg"],
-                response["messages"][0]["u"]["username"],
-                response["messages"][0]["rid"],
-                self,
+                text=response["messages"][0]["msg"],
+                user=response["messages"][0]["u"]["username"],
```

Comment:
this structure implies that rocketchat might have the concept of ids. Would need someone who knows though.

Reply:
This is correct, you can check the docs for the [channel history](https://rocket.chat/docs/developer-guides/rest-api/channels/history/).

The user id is found on: `response["messages"][0]["u"]["_id"]`